tests/unit/flask/test_charm.py

```diff
@@ -29,49 +25,46 @@ def test_flask_pebble_layer(harness: Harness) -> None:
     assert: flask charm should submit the correct flaks pebble layer to pebble.
     """
     harness.begin()
+    container = harness.charm.unit.get_container("flask-app")
```

Comment:
Same as above. The container name could be a constant defined somewhere.

Reply:
Updated, thanks!